common/util.go

```
@@ -410,3 +411,47 @@ func ReadAtMost(r io.Reader, n int) ([]byte, error) {
 	}
 	return b, err
 }
+
+func enumerateNvidiaDevices() []string {
```

Instead of logging errors in these helper functions can we return the errors and let the caller determine how to handle them (i.e. logging the errors in `livepeer.go`)? At the moment, we just end up logging these errors, but this change would give the caller the choice of how to handle the errors and doesn't require errors to always be logged in case the functions are used in other contexts.